content/source/docs/enterprise/api/ptfe_settings.html.md

```diff
@@ -0,0 +1,491 @@
+[JSON API error object]: http://jsonapi.org/format/#error-objects
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $ATLAS_TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request GET \
+  https://app.terraform.io/api/v2/admin/settings/saml
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "saml-settings",
+    "type": "settings",
+    "attributes": {
+      "enabled": true,
+      "idp_cert": "SAMPLE-CERTIFICATE",
+      "slo_target": "https://example.com/slo",
+      "sso_target": "https://example.com/sso",
+      "attr_groups": "MemberOf",
+      "attr_site_admin": "SiteAdmin",
+      "site_admin_role": "site-admins",
+      "sso_api_token_session_timeout": 1209600
+    }
+  }
+}
+```
+
+## Update SAML Settings
+`PATCH /api/v2/admin/settings/saml`
+
+-> **Note:** This endpoint cannot be accessed with [organization tokens](../users-teams-organizations/service-accounts.html#organization-service-accounts). You must access it with a [user token](../users-teams-organizations/users.html#api-tokens) which has the `site-admins` permission.
+
+Status  | Response                                         | Reason
+--------|--------------------------------------------------|-------
+[200][] | Array of [JSON API document][]s (`type: "settings"`) | Successfully updated SAML settings
+[404][] | [JSON API error object][]                    | User unauthorized to perform action
+[422][] | [JSON API error object][]                    | Malformed request body (missing attributes, wrong types, etc.)
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+[422]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422
+[JSON API document]: https://www.terraform.io/docs/enterprise/api/index.html#json-api-documents
+[JSON API error object]: http://jsonapi.org/format/#error-objects
+
+### Request Body
+
+This PATCH endpoint requires a JSON object with the following properties as a request payload.
+
+Properties without a default value are required if `data.attributes.enabled` is set to `true`.
+
+Key path                    | Type   | Default | Description
+----------------------------|--------|---------|------------
+`data.attributes.enabled`    | bool   | `false` | Allows SAML to be used. If true, all remaining attributes are required.
+`data.attributes.idp_cert`   | string |         | Identity Provider Certificate specifies the PEM encoded X.509 Certificate as provided by the IdP configuration.
+`data.attributes.slo_target` | string |         | Single Log Out URL specifies the HTTP(s) endpoint on your IdP for single logout requests. This value is provided by your IdP configuration.
+`data.attributes.sso_target` | string |         | Single Sign On URL specifies the HTTP(S) endpoint on your IdP for single sign-on requests. This value is provided by your IdP configuration.
```

Comment:
Typo: `HTTP(S)` here and `HTTP(s)` the line above.

(Do we really support HTTP for SSO URLs?)

Reply:
Will fix!
I'm not sure on the second point. Maybe @zhssh knows?